scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/util/DataValidationHelper.java

```diff
@@ -0,0 +1,133 @@
+package org.cloudfoundry.autoscaler.scheduler.util;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.List;
+import java.util.TimeZone;
+
+import org.cloudfoundry.autoscaler.scheduler.entity.ScheduleEntity;
+import org.cloudfoundry.autoscaler.scheduler.rest.model.ApplicationScalingSchedules;
+
+/**
+ * 
+ *
+ */
+public class DataValidationHelper {
+
+	/**
+	 * @param object
+	 * @return
+	 */
+	public static boolean isNotNull(Object object) {
```

Comment:
Dangerous definition to have an empty string be equivalent to a null. Normally you have an IsEmpty/IsBlank for String.